cip/CIP2016-07-07-Parameter-syntax.adoc

```diff
@@ -0,0 +1,91 @@
+= CIP2016-07-07 - Parameter syntax
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Mats Rydberg <mats@neotechnology.com>
+
+[abstract]
+.Abstract
+--
+This CIP suggests changing syntax for parameter usage, from curly braces (`{}`) to dollars (`$`).
```

Comment:
'to dollars' is imprecise (it sounds as if this was talking about the currency)... I suggest: 
', from curly braces (`{}`) to dollars (`$`).' --> 'by replacing the curly braces notation (`{}`) with the dollar delimiter (`$`).'